staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -557,6 +568,10 @@ type VirtualMachineInstanceMigrationState struct {
 	MigrationUID types.UID `json:"migrationUid,omitempty"`
 	// Lets us know if the vmi is currently running pre or post copy migration
 	Mode MigrationMode `json:"mode,omitempty"`
+	// Name of the migration policy. If string is empty, no policy is matched
+	MigrationPolicyName string `json:"migrationPolicyName,omitempty"`
```

Comment:
if the presence of a value signifies something, then consider a string pointer

Reply:
I don't think so since every policy must have a non-empty name.
I should remove the `omitempty` part

Reply:
Oh, sorry, I confused with something else.
You're absolutely right. Thanks.